actors/organizer.go

```diff
@@ -0,0 +1,407 @@
+package actors
```

Comment:
This file implements everything an organizer has to do.

The way it works is following (same for Witness):
1) Organizer.HandleWholeMessage() is called by hub
2)The received message is analyzed and another handle function is called depending on it's content. This happens recursively until we reach the "bottom level" of the message

Reply:
There is clearly some duplicated code between organizer and witness. We plan on refactoring it in the future.